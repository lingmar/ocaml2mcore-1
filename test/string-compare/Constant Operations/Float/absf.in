Float.abs 3.000000
