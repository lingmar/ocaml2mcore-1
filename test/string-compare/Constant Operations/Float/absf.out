mexpr
if leqf 0.000000 3.000000 then
  3.000000
else negf 3.000000
